embark-ui/src/containers/FileExplorerRowContainer.js

```diff
@@ -29,8 +29,16 @@ class FileExplorerRowContainer extends React.Component {
       <span className="float-right mr-2">
         {this.props.node.children &&
           <React.Fragment>
-            <a id="add-file" onClick={() => this.addFileModal.current.toggle()}><FontAwesome name="plus" className="text-success mr-2" /></a>
-            <a id="add-folder" onClick={() => this.addFolderModal.current.toggle()}><FontAwesome name="folder-open" className="text-success mr-2" /></a>
+            <span id="add-file"
+                  style={{cursor: "pointer"}}
```

Comment:
Uh oh. I'm disappointed in you young Padawan. 
It is written in the holy texts:
- One must never use inline style unless extreme conditions arise.

You can just add a utility class in `index.css` like `.pointer`

Reply:
I hope you will forgive me to taste the dark side, master, it is so appealing!
Thx for bringing me back